### PR TITLE
Prepare 1.17.1

### PR DESCRIPTION
**Pull request by smorimoto**
- #1307
- #1379

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 # Unreleased
 
+## 1.17.1
+
+- Remove exception hack (#1307)
```
Comment (ulugbekna) [inline CHANGELOG.md:7]: isn't this internal, meaning doesn't need to be in the release notes?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ocaml-platform",
   "displayName": "OCaml Platform",
-  "version": "1.17.0",
```
Comment (ulugbekna) [inline package.json:4]: I don't know what's the precedent, but shouldn't the version be 1.18.0 given we're bumping vscode engine version?

**Review by ulugbekna — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (rgrinberg) [reply package.json:4]: Are there any new features introduced by bumping the engine?

File: CHANGELOG.md (hunk 1 above)
Comment (smorimoto) [reply CHANGELOG.md]: Fair enough.

File: package.json (hunk 2 above)
Comment (smorimoto) [reply package.json:4]: There are no new features, just a bug fix, so I thought a point release would be enough.

**Review by ulugbekna — APPROVED** [review]

**Pull request merged**